incubator/virtualcluster/pkg/syncer/resources/pod/checker.go

```diff
@@ -163,6 +165,12 @@ func (c *controller) PatrollerDo() {
 		}
 		vList := listObj.(*v1.PodList)
 		for i := range vList.Items {
+			if featuregate.DefaultFeatureGate.Enabled(featuregate.SuperClusterPooling) {
+				cname, ok := vList.Items[i].GetAnnotations()[utilconstants.LabelScheduledCluster]
```

Comment:
Is is label or annotation?

Reply:
Pod scheduling result is in annotation. Maybe the const name looks a little confusing.

Reply:
yeah, that's what tripped me up.